NETPROGRAM/BasicExercise/Exercise17.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NETPROGRAM.BasicExercise
+{
+    public class Exercise17
+    {
+
+        public bool IsPrime(int n)
+        {
+
+            if(n < 2)
+            {
+                return false;
+            }
+
+            for(int i = 2; i < n/i; i++)
+            {
+                if(n % i == 0)
+                {
+                    return false;
+                }
+            }
+
+            return true;
+        }
+
+        public bool IsSymmetric(int n)
+        {
+            var number = n;
+            var sum = 0;
+
+            while(number > 0)
+            {
+                sum = sum * 10 + number % 10;
+                number /= 10;
+            }
+
+            return n == sum;
+        }
+
+        public int FindNumbers(int n)
+        {
+            var m = n;
+
+            while(!(IsPrime(m) && IsSymmetric(m)))
```

Comment:
Nên kiểm tra đối xứng trước vì thao tác đó thực hiện nhanh hơn và nó dễ ra false hơn

```suggestion
            while(!(IsSymmetric(m) && IsPrime(m)))
```

Reply:
done